content/ko/docs/topics/charts_hooks.md

```diff
@@ -0,0 +1,205 @@
+추가해야 한다.
+
+실질적으로, 이는 훅 리소스를 만드는 경우 `helm uninstall` 으로 
+리소스를 제거할 수 없음을 의미한다. 이러한 리소스를 
+삭제하려면 훅 템플릿 파일에 [사용자 지정 `helm.sh/hook-delete-policy` 어노테이션을 추가](#훅-삭제-정책)하거나 
+작업(Job) 리소스의 
+[time to live(TTL) 필드를 설정](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/)해야 
+한다.
+
+## 훅 작성하기
+
+훅은 `metadata` 섹션에 특별한 어노테이션이 있는 
+쿠버네티스 매니페스트 파일이다. 템플릿 파일이기 때문에 
+`.Values`, `.Release` 및 `.Template` 을 
+읽기를 포함한 모든 일반 템플릿 기능을 사용할 수 있다.
+
+예를 들어 `templates/post-install-job.yaml` 에 저장된 템플릿은
+`post-install` 에서 실행할 작업을 선언한다.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: post-install-job
+        image: "alpine:3.3"
+        command: ["/bin/sleep","{{ default "10" .Values.sleepyTime }}"]
+
+```
+
+다음의 어노테이션으로 이 템플릿을 훅으로 만든다.
+
+```yaml
+annotations:
+  "helm.sh/hook": post-install
+```
+
+하나의 리소스는 여러 개의 훅으로 구현할 수 있다.
+
+```yaml
+annotations:
+  "helm.sh/hook": post-install,post-upgrade
+```
+
+마찬가지로, 주어진 훅을 구현하는 다양한 리소스의 수에는 
+제한이 없다. 예를 들어 시크릿과 컨피그맵을 둘다
+pre-install 훅으로 선언할 수 있다.
+
+하위 차트에 훅이 선언되면 해당 훅도 평가된다.
+최상위 차트가 하위 차트에서 선언한 훅을 비활성화 할 수 있는 방법은 없다.
+
+결정적 실행순서를 정하는데 도움이 되는 훅에 대한 가중치를 
+정의할 수 있다. 가중치는 다음의 어노테이션을 사용하여 
+정의한다.
+
+```yaml
+annotations:
+  "helm.sh/hook-weight": "5"
+```
+
+훅 가중치는 양수여도 되고 음수여도 되지만 문자열로 표기해야
+한다. 헬름이 특정 종류의 훅 실행주기를 시작하면 해당 훅을
+오름차순으로 정렬한다.
+
+### 훅 삭제 정책
+
+대상 훅 리소스를 언제 삭제할지를 결정하는 정책을 정의할 
+수 있다. 훅 삭제 정책은 다음과 같은 어노테이션을 사용하여 
+정의한다.
+
+```yaml
+annotations:
+  "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+```
+
+한개 혹은 그 이상의 정의된 어노테이션 값을 선택할 수 있다.
+
+| 어노테이션 값              | 설명                                                                  |
+| ---------------------- | -------------------------------------------------------------------- |
+| `before-hook-creation` | 새 훅이 시작되기 전에 이전 리소스를 삭제한다. (기본값)                             |
+| `hook-succeeded`       | 훅이 성공적으로 실행된 후에 리소스를 삭제한다.                                    |
+| `hook-failed`          | 실행 중 훅이 실패한 경우 리소스를 삭제한다.                                      |
+
+훅 삭제 정책 어노테이션이 지정되지 않은 경우, 기본적으로 `before-hook-createion` 동작이
```

Comment:
`before-hook-createion` should be `before-hook-creation`

Reply:
I will update docs as your comment.